### PR TITLE
Remove noarch option in recipe file

### PR DESCRIPTION
**Pull request by RokasUrbonasQC**
The package should no longer be architecture independent. This PR should fix that.

Had to pin Windows C++ compiler to `vs2019` (default on the feedstock workflows seems to be `vs2017`) because compiling ANTLR code with older versions causes errors.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by RokasUrbonasQC** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,22 +10,21 @@ source:
...
   host:
     - pip
-    - python >=3.7
+    - python
     - setuptools
     - setuptools_scm
   run:
     - antlr-python-runtime =4.9.2
     - pyodbc >=4.0.30
-    - python >=3.7
```
Comment (ivergara) [inline recipe/meta.yaml:28]: Why did you remove the lower restriction for Python?

**Review by ivergara — COMMENTED** [review]
Thanks for catching that up!

**Comment by RokasUrbonasQC** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (ivergara) [reply recipe/meta.yaml:28]: And then, do you know how to convey a certain python version requirement then?

Comment (RokasUrbonasQC) [reply recipe/meta.yaml:28]: Not sure, I'm looking into it

**Comment by RokasUrbonasQC** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (RokasUrbonasQC) [reply recipe/meta.yaml:28]: Just added an option that should be equivalent to the restriction we had before:

https://github.com/conda-forge/pytsql-feedstock/pull/5/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR15

**Review by ivergara — APPROVED** [review]
Excellent!

**Review by kklein — APPROVED** [review]
